components/brave_wallet/browser/solana_message.cc

```diff
@@ -98,6 +98,20 @@ void SolanaMessage::GetUniqueAccountMetas(
   }
 }
 
+std::uint8_t SolanaMessage::GetNumberOfSigners() const {
+  std::vector<SolanaAccountMeta> unique_account_metas;
+  GetUniqueAccountMetas(&unique_account_metas);
+
+  std::uint8_t signers = 0;
+  for (auto account_meta : unique_account_metas) {
+    if (account_meta.is_signer) {
```

Comment:
I think we should check against UINT8_MAX here to avoid the case that num of signers > uint8_t max, which is invalid.

Reply:
Make this function return absl::optional and return absl::nullopt.

Reply:
Just in case it's useful: the latest Chromium also has base::expected: https://source.chromium.org/chromium/chromium/src/+/main:base/types/expected.h, an example in brave-core is here: https://github.com/brave/brave-core/blob/master/components/debounce/browser/debounce_rule.cc#L127